### ISSUE TITLE
When navigating in a browse dialog, or in microsoft word 2013 begin panel, nvda does not announces the status the elements.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3512]
**Reported by lallerkee1 on 2013-09-10 09:32**
When I navigate in Microsoft word 2013 beginner pane, the empty document, and templates links are not read.
This problem occurs, when I want to browse some files, such as attaching the log file.
It is in all windows a deffect.

**Comment by ehollig** [comment]
Closing, due to lack of information, and inability to contact the original poster. I also am not able to determine exactly what the issue is from the log provided.
